TCAT/Controllers/RouteDetailViewController.swift

```diff
@@ -458,6 +443,21 @@ class RouteDetailViewController: UIViewController, GMSMapViewDelegate, CLLocatio
         }
 
     }
+    
+    func setIndex(of marker: GMSMarker, with waypoint: Waypoint) {
+        marker.zIndex = {
+            switch waypoint.wpType {
+            case .bus: return 1
+            case .walk: return 1
+            case .origin: return 3
+            case .destination: return 3
+            case .stop: return 1
+            case .walking: return 0
+            case .bussing: return 999
```

Comment:
explain why you return 999 please

Reply:
The random high number worked, it's just a high constant